libs/smart-contracts/src/contracts/token-vesting.ts

```diff
@@ -14,13 +14,13 @@ export class TokenVesting {
   }
 
   stakeTokens(amount: string, vegaPublicKey: string) {
-    return this.contract.stake_tokens(amount, vegaPublicKey);
+    return this.contract.stake_tokens(amount, `0x${vegaPublicKey}`);
```

Comment:
This add 0x repeats in couple place. We should have function for this, opposite of remove0x
ie 
`const add0x = (str:string) = str.replace(/^(?!0x)(.*)/, '0x$1')`
or
`const add0x = (str:string) = !str.startsWith('0x') ? '0x${str}' : str`